pulsar-client-cpp/python/pulsar_test.py

```diff
@@ -1228,6 +1228,16 @@ def test_json_schema_encode(self):
         second_encode = schema.encode(record)
         self.assertEqual(first_encode, second_encode)
 
+    def test_garbage_collection(self):
+        oldrlimit = resource.getrlimit(resource.RLIMIT_NOFILE)
+        try:
+            resource.setrlimit(resource.RLIMIT_NOFILE, (15, 15))
+            for i in range(100):
+                client = pulsar.Client(service_url=self.serviceUrl)
+                con = client.subscribe(topic='mytopic', subscription_name=f'mysub-{i}')
```

Comment:
```suggestion
                con = client.subscribe(topic='mytopic', subscription_name='mysub-' + str(i))
```

Use a Python2 compatible way to create a string.